### DISCUSSION TITLE
邀请社区成员共同探讨 Umo Editor 项目的发展

### DISCUSSION THREAD

**Discussion by umodoc** [umodoc/editor#42] [Ideas]
**亲爱的社区成员们，**

非常感谢大家一直以来对我们项目的关注和支持。为了使我们的项目更加贴合用户需求，更好地服务于大家，我们在此发起一个讨论，希望听到来自不同角度的声音和建议。

### 讨论目的

通过这次讨论，我们希望达到以下几个目的：

1. **了解使用场景**：收集用户在哪些具体场景下使用我们的项目，以及使用过程中遇到的问题和挑战；
2. **需求调研**：探讨用户对项目功能的具体需求，包括现有功能的改进和新功能的期待；
3. **改进建议**：邀请用户提出对项目改进的具体建议，包括但不限于性能优化、用户界面(UI)/用户体验(UX)设计、新特性开发等；
4. **发展方向**：讨论项目未来的发展方向，包括技术趋势、市场定位和长远规划，以确保项目能够持续发展并满足未来的需求。

#### 使用场景

- 您是如何使用我们的项目的？
- 在使用过程中，有哪些场景特别需要我们的支持？

#### 需求调研

- 您认为我们的项目在哪些方面需要改进？
- 是否有特定的功能或特性是您希望我们添加的？

#### 改进建议

- 您对我们的项目有哪些具体的改进建议？
- 这些建议如何帮助提升项目的可用性和用户体验？

### 讨论方式

我们鼓励大家以开放和建设性的态度参与讨论。您可以：

- 在此讨论下直接回复您的想法和建议。
- 如果您有详细的方案或设计，欢迎以附件的形式分享。
- 如果您希望进一步讨论，可以发起新的议题或子话题。

### 讨论规则

- 请保持讨论的尊重和友好，避免人身攻击或负面言论。
- 尽量提供**具体、详细的反馈**，这有助于我们更好地理解您的需求。
- 如果您的想法得到了社区的广泛支持，我们将考虑将其纳入项目的开发计划。

我们期待与每一位社区成员的交流，您的每一条反馈都是我们前进的动力，感谢您的参与和支持！

The English version of this discussion can be found at #43 . If you are a non-Chinese speaker, please proceed to [Invitation to Community Members to Discuss the Development of the Umo Editor](https://github.com/umodoc/editor/discussions/43).

**Comment by pkbdemo** [comment]
插入表格的功能，有预计添加隐藏表格边框线条吗?

**Reply by umodoc** [reply]
表格功能会整体强化，包括边框颜色设置，开发计划见：https://editor.umodoc.com/cn/docs/roadmap

**Comment by xkyong** [comment]
这里提个小建议，如果当前 Discussion 评论完要手动删掉然后再创建 issue 的话，建议这里直接设置 `Lock conversation` ，让社区的同学直接转到 issue tab 中提 issue 就行了~

**Reply by umodoc** [reply]
锁定后其他人无法评论，只能提 Issue，我们希望功能相关的回复能在此讨论下进行

**Comment by ZangHao0130** [comment]
有考虑类似word修订相关功能吗，包括但不限于修订内容线上线下互通

**Reply by umodoc** [reply]
在开发计划中，见：https://editor.umodoc.com/cn/docs/roadmap

**Comment by ChenStr** [comment]
vue3 使用该组件会出现不能换页的问题希望解决一下

**Comment by lytdev** [comment]
编辑能能否作为一个普通的富文本编辑器，不需要分页使用的那种

**Reply by china-wangxu** [reply]
2025年1月15日刚发布的最新版本默认无分页特性，如果需要分页特性需要拉取单独的分支 https://github.com/umodoc/editor/tree/page-break-v4

**Comment by china-wangxu** [comment]
有没有图表的开发计划，我用echart贡献一个能否满足项目规范

**Reply by umodoc** [reply]
在不显著增加打包体积的情况下，欢迎贡献。

**Comment by wumingyue147** [comment]
协同编辑啥时候能上线？超级期待

**Comment by wusiyuan02** [comment]
有没有word导出的具体开发计划，个人觉得一个文本编译器应该是有导出word的能力

**Comment by MARIOMARUI** [comment]
prosemirror似乎对移动端不太友好，touch事件无法处理

**Comment by wellgone** [comment]
请问有开启react版分支的计划吗？

**Reply by umodoc** [reply]
暂时没有，如果你可以完成这项工作，我们愿意单独设置一个分支或者仓库，用以方便社区协作，共同完成这一壮举。
